crates/mun_compiler/src/driver.rs

```diff
@@ -107,14 +106,22 @@ impl Driver {
 
     /// Emits all diagnostic messages currently in the database; returns true if errors were
     /// emitted.
-    pub fn emit_diagnostics(&self, writer: &mut impl WriteColor) -> Result<bool, failure::Error> {
+    pub fn emit_diagnostics(&self) -> Result<bool, failure::Error> {
```

Comment:
Would it also be possible to still be able to pass a stream into this function to which the diagnostics are written.

Reply:
I use `eprintln!` macro to output diagnostics, which write diagnostics in `io::stderr`. And so you can get this stream from anywhere by `io::stderr()`. Do that make sense?

Reply:
> Would it also be possible to still be able to pass a stream into this function to which the diagnostics are written.

I use eprintln! macro to output diagnostics, which write diagnostics in io::stderr. And so you can get this stream from anywhere by io::stderr(). Do that make sense?

Reply:
I would prefer that you pass in a stream and log to that instead of assuming that you want to output to stderr. So that the API becomes something like:
```
pub fn emit_diagnostics(&self, writer: &mut std::io::Write);
```
Something like that..